docs/maintain-nominator.md

```diff
@@ -12,6 +12,36 @@ By doing so, you become a nominator for the validator of your choice. Pick your
 
 While your DOTs are staked by nominating a validator, they are 'locked' (bonded). You can receive new DOTs in your account but you cannot stake as validator or transfer DOTs away from your account. You can [un-nominate at any time](maintain-guides-how-to-unbond) to unlock your funds. Keep in mind that the un-nomination is effective in the next era.
 
+### What does active/inactive nomination mean?
+
+When you go to the [Account actions](https://polkadot.js.org/apps/#/staking/actions) under staking page, you should see your bonded accounts and nomination
+status. If not, you can follow [this](maintain-guides-how-to-nominate-kusama) guide to configure it first. Your nominations will be effective in the next era, that means roughly 6 hours.
+
+![Nominations](/img/staking/polkadotjs-staking-account-actions.jpg)
+
+Suppose you have nominated five validator candidates, but 3 out of 5 were elected to 
+be the active validators then you should see 2 of your nominations are inactive. Active nomination means your nominated validators have been elected to be the active validator, whereas inactive means not get elected. 
+
+### What does inactive nominations could affect your rewards?
```

Comment:
```suggestion
### Possible effect of inactive nominations on rewards
```

Reply:
Headings shouldn't be questions: https://writing.stackexchange.com/questions/14516/can-section-headings-in-a-paper-be-questions